src/mem/ruby/protocol/GPU_VIPER-TCP.sm

```diff
@@ -520,6 +544,20 @@ machine(MachineType:TCP, "GPU TCP (L1 Data Cache)")
     }
   }
 
+  action(ldmi_loadDoneMissInv, "ldmi",
```

Comment:
I added this action/the updated transition below because BC in Pannotia from the weekly tests fails from the assert on line 539 in l_loadDoneMiss otherwise when another request evicted this line before the response returned.  However, although this update resolves that problem, there is now a segfault with BC that I am still chasing down.

All per checkin and daily GPU tests pass with this update, as well as a number of microbenchmarks that we are running to isolate latency/bandwidth/etc. for various GPU components.  So it seems like the issues are isolated to the "pre-emptive eviction" case that the larger scale of Pannotia introduces.